### PR TITLE
New performance site

### PR DESCRIPTION
**Pull request by solveigsg12**
new look on performances and previous performances info

performances-new-look

### CODE REVIEW

File: src/reducers/previousPerformances.js [hunk 1]
```diff
@@ -1,38 +1,76 @@
...
 });
 
 export default createReducer(initialState, {
-  [types.SET_PREVIOUS_PERFORMANCE]: (state, {name, img1, img2, img3}) => {
-    return state
-      .setIn([name, 'imgUrlOne'], img1)
-      .setIn([name, 'imgUrlTwo'], img2)
-      .setIn([name, 'imgUrlThree'], img3);
+  [types.SET_PREVIOUS_PERFORMANCE]: (state, {name, images}) => {
+    console.log('hallo reducer?', images);
```
Comment (evasteingrims) [inline src/reducers/previousPerformances.js]: console!

File: src/sections/FrontPage.js [hunk 2]
```diff
@@ -7,13 +7,13 @@ class FrontPage extends Component {
...
-  componentDidMount() {
-    const video = this.iframe;
-    console.log(video, 'hvað er video??', video.player, this.refs.player);
-    if (video) {
-      video.mute();
-    }
-  }
+  // componentDidMount() {
+  //   const video = this.iframe;
+  //   console.log(video, 'hvað er video??', video.player, this.refs.player);
```
Comment (evasteingrims) [inline src/sections/FrontPage.js]: Why is this commented out? Remove?

File: src/sections/PreviousPerformanceInfo.js [hunk 3]
```diff
@@ -1,34 +1,36 @@
...
-            className="previousPerformanceInfoImgTwo"
-          />
-          <img
-            alt="this is image from event 3"
-            src={this.props.performanceObject.get('imgUrlThree')}
-            className="previousPerformanceInfoImgThree"
-          />
+          <div className="performanceInfoImgContainer">
+            {this.props.performanceObject.size &&
+              this.props.performanceObject.get('images').map((image, index) => {
```
Comment (evasteingrims) [inline src/sections/PreviousPerformanceInfo.js]: Nice one!

File: src/reducers/previousPerformances.js (hunk 1 above)
Comment (solveigsg12) [inline src/reducers/previousPerformances.js]: obbosi fixed: 	d6e5fef

File: src/sections/FrontPage.js (hunk 2 above)
Comment (solveigsg12) [inline src/sections/FrontPage.js]: true! fixed: 	d6e5fef

**Pull request merged**